assets/js/pages/HostDetailsPage/HostDetails.test.jsx

```diff
@@ -388,138 +393,105 @@ describe('HostDetails component', () => {
   });
 
   describe('operations', () => {
-    it('should run Saptune apply operation when clicking apply button in modal', async () => {
```

Comment:
I had a hard time with this test particularly. I attempted many combinations of `await act(...)`, `await waitFor(...)` but I could not make it less flaky.

I thought that:
- at the end what was being tested here is already tested in `SaptuneSolutionOperationModal` tests
- the test that was attempted here sounds more suitable for a cypress e2e test